### PR TITLE
lfe 1.2.0

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by dunn** [comment]
Upstream (now?) recommends `make install`: https://github.com/rvirding/lfe#installation

**Comment by ilovezfs** [comment]
@dunn PR refreshed.

File: Formula/lfe.rb [hunk 1]
```diff
@@ -14,12 +13,17 @@ class Lfe < Formula
...
-    prefix.install "ebin"
+    ENV.deparallelize
+    system "make"
+    system "make", "MANINSTDIR=#{man}", "install-man"
+    system "make", "emacs"
+    libexec.install "bin", "ebin"
+    bin.install_symlink (libexec/"bin").children
+    doc.install Dir["doc/*.txt"]
+    pkgshare.install "dev", "examples", "test"
+    elisp.install Dir["emacs/*.elc"]
```
Comment (dunn) [inline Formula/lfe.rb]: 😻

**Comment by ilovezfs** [comment]
@dunn this fixes the parallelization bug:
```
diff --git a/Makefile b/Makefile
index 3348017..5cecba3 100644
--- a/Makefile
+++ b/Makefile
@@ -95,7 +95,7 @@ lfec-compile: $(addprefix $(EBINDIR)/, $(LBINS))
 $(EBINDIR)/$(APP_DEF): $(SRCDIR)/$(APP_DEF).src
 	cp $(SRCDIR)/$(APP_DEF).src $(EBINDIR)/$(APP_DEF)
 
-erlc-lfec: erlc-compile lfec-compile $(EBINDIR)/$(APP_DEF)
+erlc-lfec: erlc-compile $(EBINDIR)/$(APP_DEF)
 
 emacs:
 	cd $(EMACSDIR) ; \
```

**Pull request closed without merging**